sys-kernel/dracut/files/059-kernel-install-uki.patch

```diff
@@ -0,0 +1,150 @@
+Combination of:
```

Comment:
IIRC (I tried it myself in the past, I think, when debugging something) the individual patches all apply (or nearly did). If they do, maybe just include them as 3 separate patches (or N where N is the number of commits) so we have proper attribution and it's obvious what they are?

But see what mike says, I don't feel that strongly about it now you already did it

Reply:
Yeah, I generally prefer to apply upstream patches directly where feasible. However, don't bother if that creates additional work at this point.

Reply:
> Yeah, I generally prefer to apply upstream patches directly where feasible. However, don't bother if that creates additional work at this point.

I grouped them together because they are all thematically the same. I'll keep it as is for now, but next time I'll just leave all the commits separated.